srcalc/src/main/java/gov/va/med/srcalc/domain/model/Rule.java

```diff
@@ -34,6 +34,8 @@
 @Table(name = "rule")
 public final class Rule
 {
+    public static final int MAX_MATCHERS = 10;
```

Comment:
Since we don't verify this limit anywhere in this class, how about just making the limit a UI feature and move this constant to `EditRule`?